src/colmap/exe/model.cc

```diff
@@ -121,21 +121,27 @@ std::vector<Eigen::Vector3d> ConvertCameraLocations(
     const std::vector<Eigen::Vector3d>& ref_locations) {
   if (ref_is_gps) {
     const GPSTransform gps_transform(GPSTransform::WGS84);
-    if (alignment_type != "enu") {
+    if (alignment_type != "enu" && alignment_type != "utm") {
       LOG(INFO) << "Converting Alignment Coordinates from GPS (lat/lon/alt) "
                    "to ECEF.";
       return gps_transform.EllToXYZ(ref_locations);
-    } else {
+    } else if (alignment_type == "enu") {
       LOG(INFO) << "Converting Alignment Coordinates from GPS (lat/lon/alt) "
                    "to ENU.";
       return gps_transform.EllToENU(
           ref_locations, ref_locations[0](0), ref_locations[0](1));
+    } else if (alignment_type == "utm") {
+      LOG(INFO) << "Converting Alignment Coordinates from GPS (lat/lon/alt) "
+                   "to UTM.";
+      const auto [utm, _] = gps_transform.EllToUTM(ref_locations);
```

Comment:
Should we handle the possibility of images crossing utm zones. This can happen particularly when the video is shot at the zone boundaries.